devops/jobs/CreateSandbox.groovy

```diff
@@ -94,6 +94,18 @@ class CreateSandbox {
                         relativeTargetDirectory('configuration-internal')
                     }
                 }
+                git {
```

Comment:
@coryleeio pointed out that `configuration-internal` above is actually `edx-internal` so we shouldn't be checking it out twice.

Cory:
```
I also think that configuration-internal and configuration-secure, might actually be edx-internal and
edx-secure at a different level of abstraction and that adding edx-internal there may not be necessary, 
you have to remember that code is in configuration so it’s open source so relying on edx-internal directly 
isn’t appropriate
```

@nadeemshahzad could your logic point at the `configuration-internal` folder instead?

Reply:
@coryleeio I had misunderstood this and thought that `configuration-internal` was `edx-internal`, but it is in fact https://github.com/edx/sandbox-internal so we either need to:

1: Copy the four argocd django application folders out of edx-internal and into sandbox-internal.
2: Clone edx-internal onto the box.

Reply:
@nadeemshahzad would like to try #1 first since there is less of a risk of leaking remote config data.

Reply:
That sounds good to me.